projects/igniteui-angular/src/lib/select/select.component.ts

```diff
@@ -454,7 +459,15 @@ export class IgxSelectComponent extends IgxDropDownComponent implements IgxSelec
         if (args.cancel) {
             return;
         }
-        this.scrollToItem(this.selectedItem);
+    }
+
+    /** @hidden @internal */
+    public onToggleContentAppended() {
+        if (this.selectedItem && !(this._overlayCached.positionStrategy instanceof SelectPositioningStrategy)) {
```

Comment:
While the cached object will likely work, it's kind of odd - the overlay service already has that stored with the actual values and the append event receives arguments with the Id, so if you inject the service you can get the overlay info with this id and access the position setting in use to check it. Something like:
```ts
const info = this.overlayService.getOverlayById(event.Id);
if (info?.settings?.positionStrategy instanceof SelectPositioningStrategy) return;
super.onToggleContentAppended();
```